src/App.js

```diff
@@ -266,23 +628,47 @@ const App = () => {
     return "";
   };
 
-  const filteredStats = leagueEnum
-    ? Object.values(leagueEnum).filter((stat) => stat !== "Lg")
+  const filteredStats = (gauntletMode ? gauntletLeagueEnum : leagueEnum)
+    ? Object.values(gauntletMode ? gauntletLeagueEnum : leagueEnum).filter(
+        (stat) => stat !== "Lg"
+      )
     : [];
 
   const handleCloseModal = () => {
     setIsModalOpen(false);
   };
 
+  const handleOpenModal = () => {
+    setIsModalOpen(true);
+  };
+
+  const handleOpenRulesModal = () => {
+    setIsRulesModalOpen(true);
+  };
+
+  // Add this function to close the Gauntlet Mode modal
+  const closeGauntletModal = () => {
+    setIsGauntletModalOpen(false);
+  };
+
   return (
     <div className="container" style={{ marginBottom: "100px" }}>
-      {isRulesModalOpen && (
+      <ButtonContainer>
+        <LogoButton onClick={handleOpenModal}>
+          <img src={whiteLogo} alt="Logo" />
```

Comment:
specifying `width=x` and `height=auto` would be good here, I remember it looked like it was skewing the height